h2o-genmodel/src/main/java/hex/genmodel/attributes/VariableImportances.java

```diff
@@ -36,4 +41,27 @@ protected static VariableImportances extractFromJson(final JsonObject modelJson)
 
         return new VariableImportances(varNames, relativeVarimps);
     }
+
+    /**
+     *
+     * @param n how many variables is in the output. If n >= number of variables or n <= 0 then all variables are returned.
+     * @return descending sorted array of String -> double.
+     *          Where String is variable and double is relative importance of the variable.
+     */
+    public KeyValue[] topN(int n) {
+        if (n <= 0 || n > _importances.length) {
+            n = _importances.length;
+        }
+        KeyValue[] sortedImportances = new KeyValue[_importances.length];
+        for (int i = 0; i < _importances.length; i++) {
+            sortedImportances[i] = new KeyValue(_variables[i], _importances[i]);
+        }
+        Arrays.sort(sortedImportances, new Comparator<KeyValue>() {
```

Comment:
why not use lambda syntax?

Reply:
Lambda syntax would be my first choice too but commend in build.gradle says it has to stay to stay Java 7 compatible:

https://github.com/h2oai/h2o-3/blob/6e4341278470bc544b9f6c950a42e146e83a3dc0/h2o-genmodel/build.gradle#L6

Reply:
you are right, I forgot about this